sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -500,10 +519,22 @@ public partial class IndexingParameters
     {
         public IndexingParameters() { }
         public int? BatchSize { get { throw null; } set { } }
-        public System.Collections.Generic.IDictionary<string, object> Configuration { get { throw null; } set { } }
+        public System.Collections.Generic.IDictionary<string, object> Configuration { get { throw null; } }
         public int? MaxFailedItems { get { throw null; } set { } }
         public int? MaxFailedItemsPerBatch { get { throw null; } set { } }
     }
+    public static partial class IndexingParametersExtensions
+    {
+        public static Azure.Search.Documents.Indexes.Models.IndexingParameters ExcludeFileNameExtensions(this Azure.Search.Documents.Indexes.Models.IndexingParameters parameters, params string[] extensions) { throw null; }
```

Comment:
Nit: Could we make this ExcludedFileNameExtensions?

Reply:
Since we're going to likely be defining these in swagger and will generate the properties as-is on models, I'd rather the eventual property names match the JSON schema property names. So, as samples, I'll probably be changing these related extension methods to `Set<would-be model property names>` so we don't conflict later if someone is using these samples in their code.